Library/Formula/mactelnet.rb

```diff
@@ -0,0 +1,318 @@
+class Mactelnet < Formula
+  homepage "https://github.com/haakonnessjoen/MAC-Telnet"
+  url "https://github.com/haakonnessjoen/MAC-Telnet/archive/v0.4.0.tar.gz"
+  sha1 "57f2a4ae4930e1b4d57c41dd2fafb38ca1142ec5"
+
+  depends_on "gettext"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  # MAC-Telnet 0.4.0 doesn't build against Mac OS X:
+  #   https://github.com/haakonnessjoen/MAC-Telnet/issues/13
+  #
+  # The patch below was submitted as:
+  #   https://github.com/haakonnessjoen/MAC-Telnet/pull/14
+  patch :DATA
```

Comment:
I found a fourth option: point the formula to a `.patch` of the 0.4.0-specific commit currently in the upstream PR. This is sort of like option one except it still works even if/when the PR gets updated to merge cleanly against upstream master.